etc/delphix-platform/ansible/apply

```diff
@@ -42,6 +42,42 @@ function apply_playbook() {
 	fi
 }
 
+#
+# To reduce the bootup latency and the latency to start services that
+# depend on this service, we only want to apply the configuration a
+# single time. This way, the configuration will be applied on the first
```

Comment:
The ansible roles also need to be applied on upgrade (since the ansible logic can change at that point). How will that get accomplished?

Reply:
I'll open a PR to update the upgrade-scripts to remove this file when we do an upgrade. I can make that change, and open the appliance-build PR prior to landing this; so that can be evaluated before we land any of the changes involved with this approach.

Reply:
I've opened this: https://github.com/delphix/appliance-build/pull/160